Contest_Control_System_Requirements.md

```diff
@@ -964,31 +964,101 @@ problem.
 ### Contest API
 
 It is a requirement that the CCS provide a compliant implementation of
-the [Contest API](contest_api). In addition to all required endpoints
-and elements (i.e. the minimal compliant implementation) the CCS must
-also provide the following endpoints and elements:
-
-| Endpoint         | Element                      |
-| :--------------- | :--------------------------- |
-| `/contests`      | `formal_name`                |
-| `/contests`      | `scoreboard_freeze_duration` |
-| `/contests`      | `penalty_time`               |
-| `/languages`     | `compiler`                   |
-| `/languages`     | `runner`                     |
-| `/problems`      | `rgb`                        |
-| `/problems`      | `color`                      |
-| `/problems`      | `time_limit`                 |
-| `/groups`        | `icpc_id`                    |
-| `/organizations` | All required elements.       |
-| `/organizations` | `icpc_id`                    |
-| `/organizations` | `formal_name`                |
-| `/teams`         | `icpc_id`                    |
-| `/teams`         | `display_name`               |
-| `/teams`         | `organization_id`            |
-| `/teams`         | `group_ids`                  |
-| `/judgements`    | `max_run_time`               |
-| `/runs`          | `run_time`                   |
-| `/awards`        | All required elements.       |
+the [Contest API](contest_api).
+
+The CCS must provice access to the Contest API including the 
+`/scoreboard` and `/event-feed` endpoints as well as consistent with 
+(at least) the following return from the `/access` endpoint:
+
+```json
+{
+   "capabilities": [],
+   "endpoints": [
+      { 
+         "type": "contest", 
+         "properties": [
+            "id", "name", "formal_name", "start_time", "duration", 
+            "scoreboard_freeze_duration", "penalty_time"
```

Comment:
No. Most years the CCS hasn't supported pause times, so when the CDS clears the contest start it remembers the current time as a pause, it can be manipulated outside of the CCS, and then when resumed it just sets the start time on the CCS accordingly.